sources/packages/backend/libs/utilities/src/config/config.service.ts

```diff
@@ -183,6 +184,15 @@ export class ConfigService {
     });
   }
 
+  /**
+   * Database seeding for testing configuration.
+   */
+  get testDatabaseSeeding(): TestDatabaseSeedingConfig {
```

Comment:
The configuration here should be the generic one for the DB connection.
It it is not part of the PR to create all DB properties now we should at least create this one as part of an object that would map them all.